src/c_api.c

```diff
@@ -0,0 +1,27 @@
+#define PY_SSIZE_T_CLEAN
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+
+#include <Python.h>
+#define PyGEOS_API_Module
+
+#include "c_api.h"
+#include "geos.h"
+#include "pygeom.h"
+
+// TODO: provide typedefs as opaque pointers?
+
+// TODO: provide context handle from here instead of via GEOS?
+// use GEOS_INIT / GEOS_FINISH
+
+extern char* PyGEOS_GEOS_API_Version(void) {
```

Comment:
I'd like to have some uniform naming convention for the external API. Maybe prefix all functions with `PyGEOS_` (so with underscore?)